controllers/openstackmachine_controller.go

```diff
@@ -452,11 +611,32 @@ func (r *OpenStackMachineReconciler) reconcileNormal(ctx context.Context, scope
 		return ctrl.Result{}, nil
 	}
 
+	err = r.reconcileAPIServerLoadBalancer(scope, openStackCluster, openStackMachine, instanceStatus, instanceNS, clusterName)
```

Comment:
Guessing this was to make gocyclo happy?

Reply:
Yep, initially it was possible to pass by just keeping down additions to reconcileNormal but after having to remove a couple of lines after every rebase, I just gave myself some margins 🫣